### ISSUE TITLE
Staging / Production?

### ISSUE THREAD

**Issue by aemadrid** [getkuby/kuby-core#2]
Have you thought on how to handle sending deployments to staging or production? Maybe namespaces?

**Comment by camertron** [comment]
Hey @aemadrid,

Yes, definitely :) You can specify more than one environment in your Kuby config like this:

```kuby
# old way (< v0.6)
Kuby.define(:production) do
  ...
end

Kuby.define(:staging) do
  ...
end

# new way (>= v0.6)
Kuby.define('my-app') do
  environment(:production) do
    ...
  end

  environment(:staging) do
    ...
  end
end
```

Then you can specify the Kuby environment in an environment variable when running the rake tasks, eg:

```bash
KUBY_ENV=staging bundle exec rake kuby:deploy
```

Does that answer your question?

**Comment by aemadrid** [comment]
After looking at the docs more it made sense. Although I did not know about the new app level. You might want to fix the docs for that.

**Comment by camertron** [comment]
Yes, I definitely need to do that. I just released v0.6.0 this morning, so the docs are still a little behind.

**Comment by aemadrid** [comment]
Very cool addition for sure.
